typings/index.d.ts

```diff
@@ -1,229 +1,217 @@
-declare module 'discord-giveaways' {
-    import { EventEmitter } from 'node:events';
-    import {
-        Client,
-        ColorResolvable,
-        EmojiIdentifierResolvable,
-        GuildMember,
-        Message,
-        MessageEmbed,
-        MessageEmbedOptions,
-        MessageMentionOptions,
-        MessageReaction,
-        NewsChannel,
-        PermissionResolvable,
-        Snowflake,
-        TextChannel,
-        ThreadChannel,
-        User
-    } from 'discord.js';
+import {
+    Client, ColorResolvable,
+    EmojiIdentifierResolvable, GuildMember, Message,
+    MessageEmbed,
+    MessageEmbedOptions,
+    MessageMentionOptions, MessageReaction, NewsChannel, PermissionResolvable, Snowflake, TextChannel, ThreadChannel, User
+} from 'discord.js';
+import { EventEmitter } from 'node:events';
 
-    export const version: string;
-    export class GiveawaysManager extends EventEmitter {
-        constructor(client: Client, options?: GiveawaysManagerOptions, init?: boolean);
+export const version: string;
+export class GiveawaysManager<ExtraData> extends EventEmitter {
+    constructor(client: Client, options?: GiveawaysManagerOptions, init?: boolean);
```

Comment:
Ig I should make this one also optional.  I'll do it.

Reply:
what should be optional?

Reply:
> what should be optional?

The generic type. I made it optional everywhere like if the user don't provide it it'll be `any` type but I missed here. If I don't make it optional then people who are already using it in ts will get an error. I'm out of station rn so u can do it simply by adding 

`<ExtraData = any>`.

Reply:
ah yeah I understand